SpecialRequestWiki.php

```diff
@@ -147,7 +147,7 @@ public function onSubmit( array $formData ) {
 		$farmerLogEntry = new ManualLogEntry ( 'farmer', 'requestwiki' );
 		$farmerLogEntry->setPerformer( $this->getUser() );
 		$farmerLogEntry->setTarget( $this->getTitle() );
-		$farmerLogEntry->setComment( $comment );
+		$farmerLogEntry->setComment( $reason );
```

Comment:
Should be $formData['reason'],

Reply:
@paladox I will mention that $reason is used elsewhere in the file